.pre-commit-config.yaml

```diff
@@ -47,3 +47,8 @@ repos:
     rev: v1.2.2
     hooks:
     -   id: yesqa
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: end-of-file-fixer
+        exclude: '.html$|^LICENSES/|.csv$|.txt$|.svg$'
```

Comment:
I wonder if we might actually want to exclude .py files and _only_ check others (e.g., .yml), since I think .py files will already fail black if they're missing the EOF newline?